examples/nlp/question_answering/question_answering_squad.py

```diff
@@ -329,38 +325,18 @@ def create_pipeline(
         add_time_to_log_dir=False,
     )
 
-    model_type = args.pretrained_model_name.split('-')[0]
-    model_cls = nemo_nlp.utils.DEFAULT_MODELS[model_type]['class']
-
-    if args.bert_config is not None:
-        model = model_cls(config_filename=args.bert_config)
-    else:
-        """ Use this if you're using a standard BERT model.
-        To see the list of pretrained models, call:
-        nemo_nlp.nm.trainables.huggingface.BERT.list_pretrained_models()
-        """
-        model = model_cls(pretrained_model_name=args.pretrained_model_name)
+    model = nemo_nlp.nm.trainables.get_huggingface_model(
+        bert_config=args.bert_config, pretrained_model_name=args.pretrained_model_name
+    )
 
     hidden_size = model.hidden_size
 
-    if args.tokenizer == "sentencepiece":
-        try:
-            tokenizer = nemo_nlp.data.SentencePieceTokenizer(model_path=args.tokenizer_model)
-        except Exception:
-            raise ValueError(
-                "Using --tokenizer=sentencepiece \
-                        requires valid --tokenizer_model"
-            )
-        special_tokens = nemo_nlp.utils.MODEL_SPECIAL_TOKENS[model_type]
-        tokenizer.add_special_tokens(special_tokens)
-    else:
-        tokenizer_cls = nemo_nlp.data.NemoBertTokenizer
-        tokenizer_special_tokens = nemo_nlp.utils.MODEL_SPECIAL_TOKENS[model_type]
-        tokenizer = tokenizer_cls(
-            pretrained_model=args.pretrained_model_name,
-            special_tokens=tokenizer_special_tokens,
-            bert_derivate=model_type,
+    if args.tokenizer == 'sentencepiece':
+        tokenizer = nemo.collections.nlp.data.tokenizers.huggingface_utils.get_sentence_piece_tokenizer(
+            tokenizer_model=args.tokenizer_model, pretrained_model_name=args.pretrained_model_name
```

Comment:
could you set tokenizer_model in argparse to None for consistency with other models?

Reply:
done